test/hotspot/jtreg/vmTestbase/nsk/share/jdi/EventHandler.java

```diff
@@ -320,22 +324,18 @@ public boolean eventReceived(Event event) {
         );
 
         /**
-         * This listener catches uncaught exceptions and print a message.
+         * This listener catches uncaught exceptions and prints a message.
          */
-        addListener( new EventListener() {
+        addListener(new EventListener() {
             public boolean eventReceived(Event event) {
                 boolean handled = false;
 
                 if (event instanceof ExceptionEvent &&
                     defaultExceptionRequest != null &&
-                    defaultExceptionRequest.equals(event.request())) {
-
-                    if (EventFilters.filtered(event) == false) {
-                        log.complain("EventHandler>  Unexpected Debuggee Exception: " +
-                                     (ExceptionEvent)event);
-                        defaultExceptionCaught = true;
-                    }
-
+                    defaultExceptionRequest.equals(event.request()))
+                {
```

Comment:
How would you recommend doing it? Is there a style guide that covers this (I can never find it when I need it).

Reply:
Just as it was before. 
if (event instanceof ExceptionEvent &&
     defaultExceptionRequest != null &&
      defaultExceptionRequest.equals(event.request())) {

Reply:
I find that much less readable since the 2nd and 3rd lines of the `if` expression are indented the same as the first statement that follows. Previously they had added a blank line to resolve this, but I don't like that either. You shouldn't have a blank line at the start of a compound statement block.

Reply:
Well, I am not going to insist.

Reply:
The following doc (and I have no idea how "official" it is since it is 25 years old):

https://www.oracle.com/technetwork/java/codeconventions-150003.pdf

Says to indent the 2nd and 3rd lines by 8 instead of 4.  See page 6. Does that seem ok?

Reply:
Even if it is not recommended by codestyle, I think the generic rule (for all hotspot) is to keep indentation consistent.  There are a lot of places where 4 spaces are used.  So it is fine to left 4 spaces, but you could update it to 8 if it makes code readable.